bouncer/authentication/models.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils.crypto import get_random_string
+
+
+class AuthToken(models.Model):
+    key = models.CharField(_("Key"), max_length=32, primary_key=True, editable=False)
+    user = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, related_name="auth_token", verbose_name=_("User")
+    )
+    created_at = models.DateTimeField(_("Created"), auto_now_add=True)
+
+    class Meta:
+        verbose_name = _("AuthToken")
```

Comment:
This is not needed, as Django by default uses the name of the class as verbose name and we don't use translated strings.